pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.sonarsource.parent</groupId>
     <artifactId>parent</artifactId>
```

Comment:
Shouldn't we depend on `parent-oss` instead? It's probably a question for REs

Reply:
We do depend on the `parent-oss`, the latest version of the `parent` (private one) is `59.0.214`